code/obj/machinery/computer/syndicate/podbay.dm

```diff
@@ -0,0 +1,124 @@
+// Riot computer but reflavored for the Cairngorm
+
+/obj/machinery/computer/battlecruiser_podbay
+	name = "podbay authorization"
+	icon_state = "drawbr"
+	density = FALSE
+	glow_in_dark_screen = TRUE
+	/// How many authentications are needed to release the shielding on the podbay. Set the first time someone clicks this.
+	var/auth_need = null
+	var/list/authorized
+	desc = "Controls access to the podbay. Use this when your team is ready to go."
+
+	light_r = 1
+	light_g = 0.3
+	light_b = 0.3
+
+	/// Whether or not the podbay on the Cairngorm is authorized.
+	var/authed = FALSE
+	/// How long until we take matters into our own hands.
+	var/auth_delay = 10 MINUTES
+	var/area/battlecruiser_area
+
+	New()
+		..()
+		START_TRACKING
```

Comment:
Why are `START_TRACKING` and `STOP_TRACKING` required for the authorisation computer? I don't see a `by_type` iteration anywhere?

Reply:
Another case of not knowing what it does and being too scared to remove it. Removing the tracking

Reply:
> Another case of not knowing what it does and being too scared to remove it. Removing the tracking

~Obligatory remark pertaining to cargo cult programming.~

If you are at a loss as to what a certain line of code does (or in this case macro), do feel free to ask in #imcoder. The end result is that you write code to a higher standard, and the community receives contributions of a greater quality.

Reply:
Will do. I often feel my questions are too silly to ask, but it's definitely better than PRing bad code, I get that.